spec/itunes/volume_spec.rb

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spec_helper'
+require 'itunes-client'
+
+include Itunes
+
+describe Volume do
+  let(:volume) { Itunes::Player.volume }
+  let(:level) { Fixnum }
```

Comment:
okay I've made the fixes and committed to **volume-controls** so you can find the changes [here](https://github.com/syxanash/itunes-client/tree/volume-controls), is it okay?